src/app/core/components/content-top-bar/content-top-bar.component.html

```diff
@@ -26,6 +26,18 @@
       icon="pi pi-comments"
       (click)="toggleThreadVisibility(!discussion.visible)"
     ></button>
+    <ng-container *ngIf="allowWatchingGroup$ | async">
+      <alg-watch-button
+        [ngClass]="{ 'margin-right': !(navigationNeighbors$ | async) }"
+        [route]="groupInfo.route"
+        [startWatchGroupInfo]="{
+          id: groupInfo.route.id,
+          name: groupInfo.title || '',
+          currentUserCanGrantGroupAccess: !!groupInfo.currentUserCanGrantGroupAccess
```

Comment:
These two cannot be undefined in practice if `allowWatchingGroup` is known... This should be the case "by design".
Actually, group-info should have a optional "details" property like items, which would have `name`, `currentUserCanGrantGroupAccess` and `currentUserCanWatchGroup` as non optional. This way, you would pass group info detail full or undefined.
`groupInfo$` should not be separated from `allowWatchingGroup$` at the end

Reply:
Fixed